http/play/src/test/scala/PlayTests.scala

```diff
@@ -58,7 +58,7 @@ class PlayTests extends AsyncWordSpec with Matchers {
 
     "FreeS programs can interact with a given request and used as returned values in Play actions" in {
       Action.async { request =>
-        Noop[Noop.Op].noop.map(_ => Results.Ok(request.method))
+        Noop[Noop.Op].noop.map(_ => Results.Ok(request.method)).freeS
```

Comment:
We can fix this with another implicit conversion in the play module if you are in the context of `FreeS.Par`